pkg/controller/rest/outofband/models.go

```diff
@@ -49,11 +50,12 @@ type outofbandCreateRequestResponse struct { // nolint: unused,deadcode
 type outofbandCreateInvitationRequest struct { // nolint: unused,deadcode
 	// in: body
 	Body struct {
-		Label     string        `json:"label"`
-		Goal      string        `json:"goal"`
-		GoalCode  string        `json:"goal_code"`
-		Service   []interface{} `json:"service"`
-		Protocols []string      `json:"protocols"`
+		Label              string        `json:"label"`
+		Goal               string        `json:"goal"`
+		GoalCode           string        `json:"goal_code"`
+		Service            []interface{} `json:"service"`
+		Protocols          []string      `json:"protocols"`
+		RouterConnectionID string        `json:"router_connection_id"`
```

Comment:
can have several routers

Reply:
you can’t. Create an invitation supports only one destination.

Reply:
Answered above.